vita3k/renderer/src/gl/texture.cpp

```diff
@@ -84,6 +85,7 @@ void configure_bound_texture(const SceGxmTexture &gxm_texture) {
  */
 static size_t decompress_compressed_swizz_texture(SceGxmTextureBaseFormat fmt, void *dest, const void *data, const std::uint32_t width, const std::uint32_t height) {
     int ubc_type = 0;
+    const int is_2pp = (fmt == SCE_GXM_TEXTURE_BASE_FORMAT_PVRT2BPP) || (fmt == SCE_GXM_TEXTURE_BASE_FORMAT_PVRTII2BPP);
```

Comment:
2**b**pp right?

Reply:
? 2bpp is 2pp ^^, is not right, if is 2bpp is send 1 and if is not send 0